examples/segmentation/README.md

```diff
@@ -0,0 +1,241 @@
+# Segmentation Task Example
+
+This segmentation task example will be divided into the following steps:
+
+> - \[Download Camvid Dataset\](#Download Camvid Dataset)
+> - \[Building Camvid Dataset\](#Building Camvid Dataset)
+> - \[Build a Segmentation Model\](#Build a Segmentation Model)
+> - [Train with Runner](#training-with-runner)
+
+## Download Camvid Dataset
+
+First, you should get the collated Camvid dataset on OpenDataLab to use for the segmentation training example. The official download steps are shown below.
+
+```python
+# https://opendatalab.com/CamVid
+# Configure install
+pip install opendatalab
+# Upgraded version
+pip install -U opendatalab
+# Login
+odl login
+# Download this dataset
+odl get Camvid
+```
+
+Or, after register your account, you can use the following code to download the dataset.
+
+```python
+import subprocess
+import tarfile
+
+print('Downloading dataset from https://opendatalab.com/CamVid.\n')
+print('You need register an account before downloading the dataset.\n')
+try:
+    # Install and upgrade opendatalab
+    subprocess.check_call(['pip', 'install', 'opendatalab'],
+                          stdout=subprocess.DEVNULL,
+                          stderr=subprocess.DEVNULL)
+    print('Installed opendatalab.\n')
+    subprocess.check_call(['pip', 'install', '-U', 'opendatalab'],
+                          stdout=subprocess.DEVNULL,
+                          stderr=subprocess.DEVNULL)
+    print('Upgraded opendatalab.\n')
+except subprocess.CalledProcessError as e:
+    print(f'Error: {e}')
+
+print('Please input your username and password to login to OpenDataLab.')
+# Login to OpenDataLab (Please manually enter your credentials when asked)
+subprocess.check_call(['odl', 'login'])
+
+# Download the dataset
+subprocess.check_call(['odl', 'get', 'CamVid'])
+
+# Unzip dataset
+with tarfile.open('CamVid/raw/CamVid.tar.gz.00', 'r:gz') as tar:
+    tar.extractall('data')
+    print('Extracted dataset to /data.\n')
+```
```

Comment:
```suggestion
```bash
# https://opendatalab.com/CamVid
# Configure install
pip install opendatalab
# Upgraded version
pip install -U opendatalab
# Login
odl login
# Download this dataset
odl get Camvid
# preprocess data
python preproces_data.py
```